src/main/java/nurseybook/logic/commands/ViewScheduleCommand.java

```diff
@@ -58,4 +58,5 @@ public boolean equals(Object other) {
                 || (other instanceof ViewScheduleCommand // instanceof handles nulls
                 && predicate.equals(((ViewScheduleCommand) other).predicate)); // state check
     }
+
```

Comment:
Not changed in this PR but just to check will line 50 still be changed?

Reply:
Good catch! This is an outdated comment that has already been handled long ago, but not removed yet.